test/kuttl/common/collectors/collect-workload-diagnostics.sh

```
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+# Collect namespace-scoped workload diagnostics when a KUTTL assertion fails.
+
+set -u
+
+if [[ "${KUTTL_COLLECTOR_OUTPUT:-}" == "file" ]]; then
+  exec > ${KUTTL_COLLECTOR_FILE_PATH:-../../../../kuttl-collectors-output.log} 2>&1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)